usr/bin/nxdk-configure

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/sh
+
+#FIXME: add explicit support for SDL / SDL2?
+#       https://wiki.libsdl.org/FAQLinux#how_do_i_add_sdl_to_my_project
+
+export CC=nxdk-cc
+export PKG_CONFIG=nxdk-pkg-config
+
+# Wrapper around autoconf ./configure scripts
+./configure --host=i686-windows
```

Comment:
If configure didn't require a `--host` argument, this could also be a generic nxdk-environment-wrapper script.

Also, another approach to this would be to have `nxdk-autoconf` which could add a custom config.sub to add an nxdk platform.

Fun find along those lines: https://github.com/autotools-mirror/autoconf/blame/b3eb40f9c9919e5c40762cab1196715bd8796af1/build-aux/config.sub#L1262-L1265